### PR TITLE
Ui and Vc

### PR DESCRIPTION
**Pull request by diego04**
The colour scheme is awful, and you have to disable the breakpoint command+7. Still have to auto format isbn.

### CODE REVIEW

File: TextCrunch/Background.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+//  TextCrunch
+//
+//  Created by Kevin De Asis on 2015-02-08.
+//  Copyright (c) 2015 Text Crunch. All rights reserved.
+//
+
+import Foundation
+
+
+public class BackgroundSetting {
```
Comment (derekdowling) [inline TextCrunch/Background.swift]: Is there a way to do this in the storyboard settings?

File: TextCrunch/BuyVC.swift [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+//  Copyright (c) 2015 Text Crunch. All rights reserved.
+//
+
+import Foundation
+
+
+import UIKit
+
+
+class Buy_VC: UIViewController {
```
Comment (derekdowling) [inline TextCrunch/BuyVC.swift]: Sorry, what is a VC?

**Comment by derekdowling** [comment]
Looking at our code base, I think it looks like we've agreed on the BlahController as our naming convention. It's a bit more clearer and easier to Grok. So if you wouldn't mind renaming your controllers to match that, that'd be great. i.e. Buy_VC -> PurchaseController

File: TextCrunch/BuyVC.swift (hunk 2 above)
Comment (gcoomber) [inline TextCrunch/BuyVC.swift]: Yeah, this should follow a different naming convention. eg. BuyViewController

File: TextCrunch/Background.swift (hunk 1 above)
Comment (diego04) [inline TextCrunch/Background.swift]: It depends on the background.

File: TextCrunch/BuyVC.swift (hunk 2 above)
Comment (diego04) [inline TextCrunch/BuyVC.swift]: Right I'll change that.

**Comment by diego04** [comment]
OK I'll rename these.

**Comment by derekdowling** [comment]
Out of date.

**Pull request closed without merging**